src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+            result._01 = left._01 | right._01;
+
+            return result;
+        }
+
+        /// <summary>Returns a new vector by performing a bitwise-exclusive-or operation on each of the elements in the given vectors.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The resultant vector.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator ^(Vector<T> left, Vector<T> right)
+        {
+            ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
+            Vector<T> result = default;
+
+            result._00 = left._00 ^ right._00;
+            result._01 = left._01 ^ right._01;
+
+            return result;
+        }
+
+        /// <summary>Returns a new vector whose elements are obtained by taking the one's complement of the given vector's elements.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <returns>The one's complement vector.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector<T> operator ~(Vector<T> value) => AllBitsSet ^ value;
+
+        /// <summary>Returns a boolean indicating whether each pair of elements in the given vectors are equal.</summary>
+        /// <param name="left">The first vector to compare.</param>
+        /// <param name="right">The first vector to compare.</param>
+        /// <returns>True if all elements are equal; False otherwise.</returns>
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool operator ==(Vector<T> left, Vector<T> right)
+        {
+            for (nint index = 0; index < Count; index++)
+            {
+                if (!ScalarEquals(left.GetElement(index), right.GetElement(index)))
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        /// <summary>Returns a boolean indicating whether any single pair of elements in the given vectors are not equal.</summary>
+        /// <param name="left">The first vector to compare.</param>
+        /// <param name="right">The second vector to compare.</param>
+        /// <returns>True if left and right are not equal; False otherwise.</returns>
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool operator !=(Vector<T> left, Vector<T> right) => !(left == right);
+
+        /// <summary>Reinterprets the bits of the given vector into those of another type.</summary>
+        /// <param name="value">The source vector</param>
+        /// <returns>The reinterpreted vector.</returns>
+        [Intrinsic]
+        public static explicit operator Vector<byte>(Vector<T> value)
+        {
+            ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
+            return Unsafe.As<Vector<T>, Vector<byte>>(ref value);
```

Comment:
This was previously doing `return new Vector<byte>(ref value.register);` and I've changed it to just `Unsafe.As`